Python/sysmodule.c

```diff
@@ -1264,18 +1320,22 @@ _PySys_GetSizeOf(PyObject *o)
     return (size_t)size;
 }
 
+/*[clinic input]
+sys.getsizeof
+
+    object:  object
+    default as dflt: object = NULL
```

Comment:
Argument Clinic cannot be used for this functions, because it is not possible to express the default value.